.travis.yml

```diff
@@ -94,16 +94,16 @@ jobs:
         - make ${MAKEOPTS} -C ports/unix test
         - (cd tests && MICROPY_CPYTHON3=python3 MICROPY_MICROPYTHON=../ports/unix/micropython ./run-perfbench.py 1000 1000)
 
-    # unix nanbox
+    # unix nanbox with Python2
```

Comment:
This is just for testing the build process works with python2 (which is good since it doesn't get done elsewhere?), so I'd clarify that here otherwise it might look like nanbox somehow requires this. Something like `(using Python2 to verify this is supported as well)`

Reply:
Yes, it's just for testing Python 2 support.  I'll update the comment.